.github/workflows/linux-e2e.yaml

```
@@ -3,6 +3,8 @@ name: e2e tests on Linux
 on:
   workflow_dispatch:
   push:
+    branches-ignore:
+      - 'dependabot/**'
```

I wish this wouldn't do the double-indentation, but as it is consistent with what already exists later in the file, I'll let it slip! 😸 
```suggestion
    branches-ignore:
    - 'dependabot/**'
```